app/views/accounting/search_transaction.html

```diff
@@ -7,6 +7,7 @@
     <div class="card">
         <div class="content">
           <br>
+          <form name="searchtransactionform" ng-controller="SearchTransactionController" novalidate="" rc-submit="submit()">
```

Comment:
Controller is already specified, its not required to specify controller name again. And also in rc-submit keep function name as searchTransaction